hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/ResourceTrackerService.java

```diff
@@ -264,6 +266,7 @@ protected void serviceStart() throws Exception {
         YarnConfiguration.RM_RESOURCE_TRACKER_ADDRESS,
         YarnConfiguration.DEFAULT_RM_RESOURCE_TRACKER_ADDRESS,
         server.getListenerAddress());
+    clusterId = conf.get(YarnConfiguration.RM_CLUSTER_ID, "default-cluster");
```

Comment:
"default-cluster" should be a constant somewhere.
Probably "_DEFAULT".

Reply:
Thanks @goiri , I will use YarnConfiguration.DEFAULT_RM_CLUSTER_ID